test/test_util_vcs.py

```diff
@@ -83,10 +82,18 @@ def _init_repo(tmpdir, repo_type):
 
 
 @pytest.fixture(params=("git", "hg"))
-def repo(request, hg_repo, git_repo):
+def repo(request, hg_repo, git_repo, monkeypatch, tmpdir):
+    """
+    The repo fixture depends on the session-scoped git and hg repo fixtures, and
+        copies the contents of the initialized repos to a temp folder
+    """
```

Comment:
This should improve the reliability on the tests as well, as it reduces runtime in a few minutes, I'm hoping it's less likely for pytest to have issues with sqlite